Source/SpacePort/SpacePort/Models/Ticket.cs

```diff
@@ -8,14 +8,16 @@ namespace SpacePort.Models
 {
     public class Ticket
     {
-        public int ID { get; set; }
-        [StringLength(15, ErrorMessage = "Text length should be shorter than 15 Characters")]
-        public string Person { get; set; }
-        [Column(TypeName = "decimal(10, 4)")]
```

Comment:
Nice to see that you're using attributes to create constraints in the database table.